doc/library/d3viz/index.txt

```diff
@@ -130,6 +130,19 @@ may not be perfect. In this case, you can press the ``Release node``
 button in the top-left corner to automatically arrange nodes. To reset
 nodes to their default position, press the ``Reset nodes`` button.
 
+You can also display the interactive graph inline in
+IPython using ``IPython.display.IFrame``:
+
+.. code:: python
+
+    from IPython.display import IFrame
+    d3v.d3viz(predict, 'examples/mlp.html')
+    Iframe('examples/mlp.html', width=700, height=500)
```

Comment:
With pydotprint, you can print directly like this:

```
            import theano
            v = theano.tensor.vector()
            from IPython.display import SVG
            SVG(theano.printing.pydotprint(v*2, return_image=True,
                                           format='svg'))
```

While working on this, do you want to implement that? I don't know that part well, but I think it should be easy. We probably need just to allow the file "str" to d3viz() as in debugprint() and there is probably a way to pass the data as a string to Iframe.